src/schema/resolvers.js

```diff
@@ -172,6 +172,35 @@ module.exports = {
         cursor.skip(skip);
       }
       return await cursor.toArray();
+    },
+
+    syncInfo: async (root, {}, {mongo: {Blocks}}) => {
+      let options = {
+        "limit": 1,
+        "sort": [["blockNum", 'desc']]
+      }
+      let syncBlockNum = null;
+      let blocks;
+      try {
+        blocks = await Blocks.find({}, options).toArray();
+      } catch(err){
+        console.error(`Error query latest block from db: ${err.message}`);
+      }
+
+      if(blocks.length > 0){
+        syncBlockNum = blocks[0].blockNum;
+      }
+
+      let chainBlockNum = null;
+      try {
+       let resp = await fetch('https://testnet.qtum.org/insight-api/status?q=getInfo');
```

Comment:
If you don't plan to update a variable, make it const.

Reply:
Correct me if I am wrong, I take `const` more like a variable for re-use and highlighted. `let` is block scoped so it is cheaper than `var` to use it in small bracket like this

Reply:
'let' is the old 'var' in es6 syntax. 'const' disallow a var to be reassigned. The choice is based on intention of the variable. You can won't fix this if you don't want to change.